src/renderer/components/RecentProjectsBlock.tsx

```diff
@@ -1,36 +1,121 @@
-import { Box, Typography, styled, colors } from '@mui/material';
-import { useSelector } from 'react-redux';
-import { ApplicationStore } from '../store/helpers';
-import GenericSquareBox from './GenericSquareBox';
+import { Box, Typography, styled, Stack, Grid, Paper } from '@mui/material';
+import { useDispatch, useSelector } from 'react-redux';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { projectOpened, pageChanged } from 'renderer/store/actions';
+import { ApplicationPage, ApplicationStore } from '../store/helpers';
+import colors from '../colors';
+import { formatDate } from '../util';
+import exampleThumbnail from '../../../assets/example-thumbnail.png';
 
 const RecentProjectsBox = styled(Box)`
-  background: ${colors.grey[900]};
-  width: calc(100vw - 80px);
+  width: calc(100vw - 40px);
```

Comment:
The recent project listings and new project box should be the same width.

Reply:
Done